### PR TITLE
Nightwatch basic structure

### PR DESCRIPTION
**Pull request by oanesbogdan**
Basic structure for nightwatch, functional testing

### CODE REVIEW

File: nightwatch/configuration/globals.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (fkroockmann) [inline nightwatch/configuration/globals.json]: Can you rename this file to nightwatch/configuration/globals.json.dist

**Comment by fkroockmann** [comment]
Very good job, how selenium is installed ?

**Comment by oanesbogdan** [comment]
I made the adjustments.

File: package.json [hunk 2]
```diff
@@ -28,6 +28,7 @@
     "jsclass": "4.0.5",
     "strip-ansi": "^2.0.1",
     "supports-color": "^1.3.1",
-    "es5-shim": "4.1.13"
+    "es5-shim": "4.1.13",
+    "nightwatch": "~0.8.16"
```
Comment (fkroockmann) [inline package.json]: add grunt-nightwatch plz

Comment (fkroockmann) [inline package.json]: add grunt-nightwatch plz

File: README.md [hunk 3]
```diff
@@ -112,6 +119,12 @@ bower install
 grunt
 ```
 
+## Run the functional testing :
+Befor running the functional testing you need to create a configuration file inside nightwatch/configuration/globals.json (The file golobals.json.dist can be used as an example)
```
Comment (fkroockmann) [inline README.md]: Before miss "e"

**Comment by fkroockmann** [comment]
Good job @oanesbogdan

**Pull request merged**